content/hi/docs/reference/glossary/daemonset.md

```diff
@@ -0,0 +1,19 @@
+---
+title: डेमनसेट (DaemonSet)
+id: daemonset
+date: 2018-04-12
+full_link: /docs/concepts/workloads/controllers/daemonset
+short_description: >
+  सुनिश्चित करें कि पॉड की प्रतिकृति क्लस्टर में नोड्स के एक सेट पर चल रही है।
+aka:
+tags:
+  - fundamental
+  - core-object
+  - workload
+---
+
+सुनिश्चित करता है {{<glossary_tooltip text="पॉड" term_id="pod" >}} की प्रतिकृति {{<glossary_tooltip text="क्लस्टर " term_id="cluster" >}} में नोड्स के एक सेट पर चल रही है।
```

Comment:
```suggestion
यह सुनिश्चित करता है कि {{< Glosary_tooltip text="क्लस्टर" term_id="cluster" >}} में नोड्स के सेट पर {{< Glosary_tooltip text="पॉड" term_id="pod" >}} की एक प्रतिलिपि चल रही है।
```

Reply:
Shouldn't it be `glossary_tooltip` instead of  `Glosary_tooltip` ?

Reply:
I'm also thinking about the same.